client/components/domains/domain-product-price/index.jsx

```diff
@@ -206,4 +214,6 @@ export default connect( ( state ) => ( {
 	domainsWithPlansOnly: getCurrentUser( state )
 		? currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY )
 		: true,
+	isCurrentPlan100YearPlan:
+		getSitePlanSlug( state, getSelectedSiteId( state ) ) === PLAN_100_YEARS ?? false,
```

Comment:
```suggestion
		getSitePlanSlug( state, getSelectedSiteId( state ) ) === PLAN_100_YEARS,
```

Nit: Would that do the same thing? Since the left operand is always `bool`, I believe we can remove the `??` part.

Reply:
Thanks! That's a good point 👍